ovn-fake-multinode-utils/playbooks/pull-ovn-tester.yml

```diff
@@ -0,0 +1,9 @@
+- name: Install ovn-tester
+  hosts: ovn_hosts
+  tasks:
+    - name: Pull latest containers
+      when: ovn_tester is defined
+      shell: |
+              set -e
+              docker pull {{ registry_node }}:5000/ovn/ovn-tester
+              docker tag {{ registry_node }}:5000/ovn/ovn-tester ovn/ovn-tester
```

Comment:
Indentation seems a bit off.
And why we need to tag the container here?  Should this be done after the build instead?

Reply:
I'll double-check the indentation, but the comment about tagging after the build doesn't make sense to me. In the `install()` function in `do.sh`, we call:
````
install_ovn_tester
pull_ovn_tester
````
The first line builds the tester. The second calls the `pull-ovn-tester.yml` ansible playbook.

Reply:
I used the installation of ovn-multinode as an example here.  In `do.sh`, `install_ovn_fake_multinode` tags and pushes the `ovn-multi-node` image. Then `pull_ovn_fake_multinode` uses the `pull-fake-multinode.yml` playbook, on relevant hosts to pull and tag the ovn-multinode image.

To be honest, this struck me as odd, too, but since it's being done with ovn-multinode, and my focus is more on using ovsdbapp, I just mimicked the behavior with ovn-tester as well.